lib/src/client/dio_interceptor.dart

```diff
@@ -32,6 +32,8 @@ class RequestsSignatureInterceptor extends Interceptor {
   ///
   /// Optionally, you can provide custom implementations for
   /// [signatureBodySourceBuilder] and [signatureBodySigner].
+  ///
+  /// Optionally, you can provide a custom loigc to get the current time via [getTime].
```

Comment:
logic

Reply:
Funny enough. I though I fixed it before publishing that PR :S